server/etcdserver/api/v3rpc/watch.go

```diff
@@ -272,6 +272,22 @@ func (sws *serverWatchStream) recvLoop() error {
 				// support  >= key queries
 				creq.RangeEnd = []byte{}
 			}
+			if creq.StartRevision < 0 {
```

Comment:
Revision should not be negative value in system. So it's to address existing issue not breaking change, from my perspective.